pkg/alerts/alertmanager/client.go

```diff
@@ -0,0 +1,121 @@
+package alertmanager
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"path"
+	"time"
+)
+
+const (
```

Comment:
Did you consider using the alertmanager cli go module instead?
Would it be more complex to maintain?